### PR TITLE
Refactor/login page

### PR DESCRIPTION
**Pull request by Jeysieeeee**

### CODE REVIEW

File: screens/auth/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Text, View, Image } from 'react-native';
+import Input from '../../components/InputField';
```
Comment (edwinpergis) [inline screens/auth/Login.tsx:3]: change it to InputFiled

**Review by edwinpergis — CHANGES_REQUESTED** [review]

File: screens/auth/Login.tsx (hunk 1 above)
Comment (edwinpergis) [inline screens/auth/Login.tsx]: change it to InputField

**Review by edwinpergis — CHANGES_REQUESTED** [review]

**Review by edwinpergis — APPROVED** [review]

File: App.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { Text, View, Image, Linking } from 'react-native';
+import OnBoarding from './screens/auth/OnBoarding';
+import Login from './screens/auth/Login'
+
+export default function App() {
+  return (
+    <View style={{ flex: 1, padding: 10 }}>
+      {/* <OnBoarding /> */}
```
Comment (carlofurio-swr) [inline App.tsx:9]: Remove unused codes

**Review by carlofurio-swr — DISMISSED** [review]

File: Style.tsx [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+    fontWeight: 'bold',
+    fontSize: 20,
+    color: '#FFFFFF',
+  },
+  forgetPasswordCointainer: {
+    paddingTop: 15,
+    flexDirection: 'row',
+  },
+  //STYLE FOR NEWS FEED
+  ScrollView: {
```
Comment (carlofurio-swr) [inline Style.tsx]: Always use camelcase for naming styles in stylesheet. Same goes for the other names below.

File: screens/auth/Login.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    <View style={styles.loginContainer}>
+      <Image
+        source={{ uri: 'https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Twitter-logo.svg/1245px-Twitter-logo.svg.png' }}
+        style={styles.TwitterLogo}
+      />
+      <Text style={styles.logintoTwitterStyle}>Log in to Twitter</Text>
+
+      <InputField
+        placeholder={'Username'}
+        onChangeText={function (text: string): void {
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx]: change it to arrow function.

File: screens/auth/Login.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+          throw new Error('Function not implemented.');
+        }} />
+      <InputField
+        placeholder={'Password'}
+        onChangeText={function (text: string): void {
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx]: also here

**Review by carlofurio-swr — CHANGES_REQUESTED** [review]

File: screens/auth/Login.tsx [hunk 6]
```diff
@@ -9,18 +9,19 @@ const Login = () => {
...
+        style={styles.twitterLogo}
       />
       <Text style={styles.logintoTwitterStyle}>Log in to Twitter</Text>
 
       <InputField
         placeholder={'Username'}
-        onChangeText={function (text: string): void {
+        onChangeText={(text)=>
+          {
           throw new Error('Function not implemented.');
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx:20]: remove this throw error, save the value on the states instead and use it as a value

File: screens/auth/Login.tsx [hunk 7]
```diff
@@ -9,18 +9,19 @@ const Login = () => {
...
         }} />
       <InputField
         placeholder={'Password'}
-        onChangeText={function (text: string): void {
+        onChangeText={(text)=> {
           throw new Error('Function not implemented.');
```
Comment (carlofurio-swr) [inline screens/auth/Login.tsx:25]: same here

**Review by carlofurio-swr — CHANGES_REQUESTED** [review]

**Pull request closed without merging**